setup.py

```diff
@@ -3,7 +3,7 @@
 setup(
     name="PySwitchbot",
     packages=["switchbot", "switchbot.devices", "switchbot.adv_parsers"],
-    install_requires=["async_timeout>=4.0.1", "bleak>=0.17.0", "bleak-retry-connector>=2.9.0"],
+    install_requires=["async_timeout>=4.0.1", "bleak>=0.17.0", "bleak-retry-connector>=2.9.0", "pycryptodome>=3.16.0"],
```

Comment:
Home assistant uses the cryptography instead. Can you switch this out?

Reply:
Done